index.html

```diff
@@ -1,11 +1,12 @@
 <!doctype html>
 <!--
+@license
 Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```

Comment:
My understanding of the policy is that we set the license at file creation time, so I'm keeping the dates in place.